gdrive/settings.py

```diff
@@ -13,6 +13,7 @@
 DEBUG = os.getenv("DEBUG", "False") == "True"
 
 LOG_LEVEL = os.getenv("LOG_LEVEL", logging.getLevelName(logging.INFO))
+logging.basicConfig(level=LOG_LEVEL)
```

Comment:
This is already set in main.

Reply:
Settings is imported before LOG_LEVEL is set in main, causing some log messages to be missed.